detox/android/detox/src/testFull/java/com/wix/detox/reactnative/idlingresources/AsyncStorageIdlingResourceTest.kt

```diff
@@ -0,0 +1,248 @@
+package com.wix.detox.reactnative.idlingresources
+
+import androidx.test.espresso.IdlingResource
+import com.facebook.react.bridge.NativeModule
+import com.wix.detox.UTHelpers.yieldToOtherThreads
+import com.wix.detox.reactnative.idlingresources.storage.AsyncStorageIdlingResource
+import com.wix.detox.reactnative.idlingresources.storage.SerialExecutorReflected
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.kotlin.any
+import org.mockito.kotlin.argumentCaptor
+import org.mockito.kotlin.eq
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.never
+import org.mockito.kotlin.times
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+import org.robolectric.RobolectricTestRunner
+import java.util.concurrent.Executor
+import java.util.concurrent.Executors
+
+private class AsyncStorageModuleStub : NativeModule {
```

Comment:
Get we get rid of Spek altogether, then?

Reply:
Still have a few spec tests